### DISCUSSION TITLE
Workarounds for using sktime on colab before the hotfix is released

### DISCUSSION THREAD

**Discussion by Stellini101** [alan-turing-institute/sktime#7079] [Q&A] [answered]
Is there a way to use sktime on colab before the hotfix is released? 
The issue: https://github.com/sktime/sktime/issues/7056
The commit that fix this issue: https://github.com/sktime/sktime/commit/e4d60d1f539cc5f5f4e4054266b9f7db4cf321c1

**Comment by fkiraly** [comment]
`pip install` a higher version of `polars`. The hotfix should be released today though.

**Reply by fkiraly** [reply, marked as answer]
FYI @Stellini101, the new release 0.32.4 should have no problems with colab anymore.

**Comment by fkiraly** [comment]
btw, @Stellini101, you wouldn't know how one can test in GitHub that a given notebook runs on colab? They seem to have a range of things installed, and sometimes old version of packages.

The point I'm stuck is, you would need to be able to spin up a VM with the same environment or dependencies, and it is unclear how to get that except through `pip list` directly in colab, which is not a GHA-ble step. More precisely, we could copy that, but that would be valid only until the next update of colab environments.
